internal/resources/diffValidation/resourse_router.go

```diff
@@ -52,11 +53,18 @@ func (r *Router) validateTier0Config() error {
 
 	//BGP graceful restart timers cannot be updated when BGP config is enabled
 	bgpEnabledPath := "tier0_config.0.bgp.0.enable_bgp"
-	if r.diff.HasChange(bgpEnabledPath) || r.diff.HasChange("tier0_config.0.bgp.0.restart_time") || r.diff.HasChange("tier0_config.0.bgp.0.stale_route_time") {
+	if r.diff.HasChange(bgpEnabledPath) || (r.diff.HasChange("tier0_config.0.bgp.0.restart_time") || r.diff.HasChange("tier0_config.0.bgp.0.stale_route_time")) {
```

Comment:
make variable restart_time and stale_route_time as constants so that the code becomes more readible

Reply:
Done